src/matlab/vigraDistance.cpp

```diff
@@ -170,13 +170,13 @@ void vigraMexFunction(vigra::matlab::OutputArray outputs, vigra::matlab::InputAr
     }
 }
 
-/*+++++++++++++++++++++++MexEntryFunc++++++++++++++++++++++++++++++++*/
```

Comment:
can you please undo these purely stylist changes.

```
git checkout master -- src/matlab/vigraDistance.cpp
git checkout master -- src/matlab/vigraResize2.cpp
git checkout master -- src/matlab/vigraResize3.cpp
```

Reply:
It's no stylist changes - the original version creates compiler warnings

Reply:
but ok - if you insist I will...

Reply:
the mex compiler doesn't like multiline comments? i'm maybe out of touch. sorry.

Can you copy paste the warning again?

Reply:
RIght, I should have listed the warning in the first place. These warnings appear for almost every .cpp file within the Matlab folder



vigraDistance.cpp
compiling: vigraDistance.cpp
mex -O -I../../include  -outdir '.' ./vigraDistance.cpp
Building with 'Xcode Clang++'.
vigraDistance.cpp:175:1: warning: '/*' within block comment [-Wcomment]
/* if a certain class is NOT supported - you will have to copy the
^
vigraDistance.cpp:176:1: warning: '/*' within block comment [-Wcomment]
/* body of the callMexFunctor function and edit it here.
^
vigraDistance.cpp:177:1: warning: '/*' within block comment [-Wcomment]
/* Supports (u)int[8|16|32|64], float and double.
^
vigraDistance.cpp:178:1: warning: '/*' within block comment [-Wcomment]
/*+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*/
^
4 warnings generated.

MEX completed successfully.
creating: vigraDistance.m

Reply:
ohhhh i see, they were opening a comment `/*` many times within the comment itself.

Reply:
Thank you for the fix.

Reply:
hi Marc @hmaarrfk ! Can we merge this fix into master?

Reply:
yes. i was looking at this last night. hsould be ok.

Reply:
thank you for your patience